lib/ansible/modules/system/ufw.py

```diff
@@ -348,7 +348,8 @@ def ufw_version():
     commands = dict((key, params[key]) for key in command_keys if params[key])
 
     if len(commands) < 1:
-        module.fail_json(msg="Not any of the command arguments %s given" % commands)
+        msg = "You must provide one these commands '{}'".format("','".join(command_keys))
```

Comment:
Maybe pass `required_one_of` parameter to `AnsibleModule` instead?

Reply:
Thanks for the suggestion :) I update my PR in this way !